### PR TITLE
Fix/update metadata

### PR DESCRIPTION
**Pull request by w-aldrich**

### CODE REVIEW

File: ingestors/kafka/src/test/scala/hydra/kafka/programs/CreateTopicProgramSpec.scala [hunk 1]
```diff
@@ -421,6 +424,45 @@ class CreateTopicProgramSpec extends AnyWordSpecLike with Matchers {
...
+          metadata
+        )
+        _ <- createTopicProgram.createTopic(subject, request, TopicDetails(1, 1))
+        _ <- metadata.addToMetadata(subject, request)
+        metadataMap <- publishTo.get
+        _ <- createTopicProgram.createTopic(subject, updatedRequest, TopicDetails(1, 1))
+        updatedMap <- publishTo.get
+      } yield {
+        val deprecatedDate = metadataMap.get(metadataTopic).toString.split("deprecatedDate")(1).substring(4).split(",")(0)
+        val updatedDate = updatedMap.get(metadataTopic).toString.split("deprecatedDate")(1).substring(4).split(",")(0)
```
Comment (lewisjkl) [inline ingestors/kafka/src/test/scala/hydra/kafka/programs/CreateTopicProgramSpec.scala:461]: Would you also be able to do `updatedMap.get(metadataTopic).get.deprecatedDate`? Or something like that? It is hard to follow these expressions here without knowing what 1, 4, etc mean.

File: ingestors/kafka/src/test/scala/hydra/kafka/programs/CreateTopicProgramSpec.scala [hunk 2]
```diff
@@ -421,6 +424,45 @@ class CreateTopicProgramSpec extends AnyWordSpecLike with Matchers {
       } yield topic shouldBe defined).unsafeRunSync()
     }
 
+    "ingest updated metadata into the metadata topic - verify deprecated date if supplied is not overwritten" in {
```
Comment (lewisjkl) [inline ingestors/kafka/src/test/scala/hydra/kafka/programs/CreateTopicProgramSpec.scala:427]: Should we also have test cases for when no deprecated date is supplied? To make sure it is overwritten? Or maybe this test case already does that and I am just not seeing it

Comment (w-aldrich) [reply ingestors/kafka/src/test/scala/hydra/kafka/programs/CreateTopicProgramSpec.scala:427]: Added

File: ingestors/kafka/src/test/scala/hydra/kafka/programs/CreateTopicProgramSpec.scala (hunk 1 above)
Comment (w-aldrich) [reply ingestors/kafka/src/test/scala/hydra/kafka/programs/CreateTopicProgramSpec.scala]: Fixed.

File: ingest/src/test/scala/hydra/ingest/http/IngestionEndpointSpec.scala [hunk 3]
```diff
@@ -81,6 +81,8 @@ final class IngestionEndpointSpec
       _ <- schemaRegistry.registerSchema("my_topic-value", otherSchema)
       _ <- schemaRegistry.registerSchema("exp.blah.blah-value", simpleSchema)
       _ <- schemaRegistry.registerSchema("exp.blah.blah-key", simpleSchema)
```
[the remark this answers is not in the document]
Comment (w-aldrich) [reply ingest/src/test/scala/hydra/ingest/http/IngestionEndpointSpec.scala]: Removed

**Review by lewisjkl — APPROVED** [review]

**Review by znickle24 — APPROVED** [review]
